### PR TITLE
Issue/11/go getable

### PR DESCRIPTION
**Pull request by yonidavidson**
Have a look,
I just need to put the artifact in place.

### CODE REVIEW

**Comment by tebeka** [comment]
Looks OK, however please fix `make circleci`

File: Makefile [hunk 1]
```diff
@@ -76,3 +77,11 @@ wtr:
 
 gdb-wtr: wtr
 	gdb wtr
+
+artifact: libcarrow.a gen artifact-linux-x86_64
+
+artifact-linux-x86_64:
```
Comment (tebeka) [inline Makefile]: IMO this rule should depend on `libcarrow.a` and not `artifact` rule

File: gen.go [hunk 2]
```diff
@@ -50,7 +50,9 @@ package carrow
 
 /*
 #cgo pkg-config: arrow plasma
-#cgo LDFLAGS: -lcarrow -L.
+#cgo LDFLAGS: -lcarrow
+#cgo linux LDFLAGS: -L./bindings/linux-x86_64
+#cgo darwin LDFLAGS: -L./bindings/darwin-x86_64
```
Comment (tebeka) [inline gen.go]: I we don't have darwin bindings - let's delete this line until we have and add an issue

File: plasma/plasma.go [hunk 3]
```diff
@@ -13,7 +13,9 @@ import (
 
 /*
 #cgo pkg-config: arrow plasma
-#cgo LDFLAGS: -lcarrow -L..
+#cgo LDFLAGS: -lcarrow
+#cgo linux LDFLAGS: -L../bindings/linux-x86_64
+#cgo darwin LDFLAGS: -L../bindings/darwin-x86_64
```
Comment (tebeka) [inline plasma/plasma.go]: Same as above comment - delete & issue

File: carrow.go [hunk 4]
```diff
@@ -9,7 +9,9 @@ import (
 
 /*
 #cgo pkg-config: arrow plasma
-#cgo LDFLAGS: -lcarrow -L.
+#cgo LDFLAGS: -lcarrow
+#cgo linux LDFLAGS: -L./bindings/linux-x86_64
+#cgo darwin LDFLAGS: -L./bindings/darwin-x86_64
```
Comment (tebeka) [inline carrow.go]: Since we don't have darwin build - remove & create issue

File: Makefile (hunk 1 above)
Comment (yonidavidson) [reply Makefile]: Good idea

File: carrow.go (hunk 4 above)
Comment (yonidavidson) [reply carrow.go]: #44

File: plasma/plasma.go (hunk 3 above)
Comment (yonidavidson) [reply plasma/plasma.go]: #44

File: gen.go (hunk 2 above)
Comment (yonidavidson) [reply gen.go]: #44

**Comment by yonidavidson** [comment]
solves #11

**Pull request merged**